portfolio/src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -33,18 +33,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/user";
+      String urlToRedirectToAfterUserLogsOut = "/";
```

Comment:
Will this just take user "home" after they logout?

Reply:
yes I think `/` would send them back to the home page.  Another option would be to send them to `/index.html` explicitly.